docs/source/optimizations/cq/cq_compile_qss.ipynb

```diff
@@ -21,26 +21,57 @@
     "</a>"
```

Comment:
<p>same comments as <code>css</code> notebook:</p><ul><li>make in-line code for qiskit-superstaq, qss.superstaq_provider.SuperstaQProvider(api_key="token")</li><li class="ql-indent-1">remove <code>api_key="token"</code></li><li>ie --&gt; i.e.</li><li>link to environment variable setup page</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/527/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
The environment variable set up link is to the `css` page and should be the `qss` page